apps/services/personal-representative/src/app/modules/personalRepresentative/e2e/personalRepresentative.spec.ts

```diff
@@ -170,11 +171,11 @@ describe('PersonalRepresentativeController', () => {
 
       // Test get personal rep
       const response = await server
-        .get(`/v1/personal-representative/all/`)
+        .get(`/v1/personal-representative/all/false?limit=1`)
```

Comment:
Ah, just noticed this URL structure now.

We should follow our API Design Guide when it comes to URL [structure](https://docs.devland.is/technical-overview/api-design-guide/resource-oriented-design#resources) and [naming](https://docs.devland.is/technical-overview/api-design-guide/naming-conventions#uris), especially since this is a Island.is API which we're sharing with other parties (difficult to "fix" later).

You can check out auth-public-api for inspiration, which is modeling similar things and was recently refactored to follow the design guide. You can also chat with @saevarma for input on specific challenges in the API design.

Reply:
E.g. we should be able to use optional or required query parameters to replace some of these URL segments.